devel/include-what-you-use/Portfile

```diff
@@ -16,14 +16,15 @@ description         A tool for use with clang to analyze #includes in C and C++
 
 long_description    {*}${description}
 
-checksums           rmd160  70834a92aaed0d19e1c345191f48279b5d1f360c \
-                    sha256  a472fe8587376d041585c72e5643200f8929899f787725f0ba9e5b3d3820d401 \
-                    size    776263
+checksums           rmd160  1a86b5f12e8cc1f09c03784cfab4f51c57381191 \
+                    sha256  34c7636da2abe7b86580b53b762f5269e71efff460f24f17d5913c56eb99cb7c \
+                    size    796681
 
-set llvm_version    17
+set llvm_version    18
 set llvm_dir        ${prefix}/libexec/llvm-${llvm_version}
 
-depends_lib-append  port:clang-${llvm_version}
+depends_build-append    port:python312
```

Comment:
How does this build know to use this version of python? Usually you have to tell it using a configure flag, environment variable, or similar.

Reply:
The build is using `find_package(Python3 COMPONENTS Interpreter)` in CMake, which manages to find it.

Reply:
The point is, it might find a different version of python if the user has it installed. You should explicitly tell the build to use ${prefix}/bin/python3.12 since that's what you've declared the dependency on.

Reply:
True, updated.